tests/parametric/test_otel_span_methods.py

```diff
@@ -484,29 +484,63 @@ def test_otel_span_reserved_attributes_overrides(self, test_agent, test_library)
-    @missing_feature(context.library == "nodejs", reason="Not implemented")
     @missing_feature(context.library == "dotnet", reason=".NET dosn't treat 'something-else' as a valid input here.")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
     @missing_feature(context.library == "python", reason="Not implemented")
     @missing_feature(context.library == "python_http", reason="Not implemented")
     @pytest.mark.parametrize(
         "analytics_event_value,expected_metric_value",
-        [
-            ("true", 1),
-            ("TRUE", 1),
-            ("True", 1),
-            ("false", 0),
-            ("False", 0),
-            ("FALSE", 0),
-            ("something-else", 0),
-            (True, 1),
-            (False, 0),
-        ],
+        [("true", 1), ("TRUE", 1), ("True", 1), ("false", 0), ("False", 0), ("FALSE", 0), (True, 1), (False, 0),],
+    )
+    def test_otel_span_basic_reserved_attributes_overrides_analytics_event(
+        self, analytics_event_value: Union[bool, str], expected_metric_value: Union[int, None], test_agent, test_library
+    ):
+        """
+            Tests the analytics.event reserved attribute override with basic inputs
+        """
+        run_otel_span_reserved_attributes_overrides_analytics_event(
+            analytics_event_value=analytics_event_value,
+            expected_metric_value=expected_metric_value,
+            test_library=test_library,
+            test_agent=test_agent,
+        )
+
+    @missing_feature(context.library == "java", reason="The expected metric value is zero")
+    @missing_feature(context.library == "golang", reason="The expected metric value is zero")
+    @missing_feature(context.library == "ruby", reason="The expected metric value is zero")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @missing_feature(context.library == "python_http", reason="Not implemented")
+    @pytest.mark.parametrize(
+        "analytics_event_value,expected_metric_value", [("something-else", None), ("fAlse", None), ("trUe", None),],
+    )
+    def test_otel_span_strict_reserved_attributes_overrides_analytics_event(
+        self, analytics_event_value: Union[bool, str], expected_metric_value: Union[int, None], test_agent, test_library
+    ):
+        """
+            Tests that the analytics.event reserved attribute override doesn't set the _dd1.sr.eausr metric
+            for inputs that aren't accepted by strconv.ParseBool
+        """
+        run_otel_span_reserved_attributes_overrides_analytics_event(
+            analytics_event_value=analytics_event_value,
+            expected_metric_value=expected_metric_value,
+            test_library=test_library,
+            test_agent=test_agent,
+        )
+
+    @missing_feature(context.library == "java", reason="Choose to not implement Go parsing logic")
```

Comment:
If the reason is really an educated choice, then you should decorate the test with `@irrelevant`, so it won't show up as not. implemented in the feature parity dashbaord.

Reply:
Thanks, I didn't know! Changed it 😃